stats/view/example_test.go

```diff
@@ -27,17 +27,15 @@ func Example_view() {
 		log.Fatal(err)
 	}
 
-	v, err := view.New(
+	v := view.New(
```

Comment:
Why do we need view.New? We should instead allow users to depend on the struct literal.